mindbender/maya/lib.py

```diff
@@ -416,6 +416,56 @@ def apply_shaders(relationships):
             cmds.sets(meshes, forceElement=shader)
 
 
+def displacement_shader_dir(nodes):
+
+    valid_nodes = cmds.ls(nodes,
+                          long=True,
+                          recursive=True,
+                          showType=True,
+                          objectsOnly=True,
+                          type="transform")
+
+    meshes = {}
+    for mesh in valid_nodes:
+        shapes = cmds.listRelatives(valid_nodes[0],
+                                    shapes=True,
+                                    fullPath=True) or list()
+
+        if shapes:
+            shape = shapes[0]
+            if not cmds.nodeType(shape):
```

Comment:
Thanks for you input BigRoy, I will read up on the "objectType" command.
Thanks for the feedback.